api/routes/openAPI/index.js

```diff
@@ -120,6 +120,10 @@ module.exports = {
                 description:
                   'When the contractor resource will end work; date only'
               },
+              totalCost: {
+                type: 'string',
```

Comment:
This will change to `number` in #1154 - pretty silly that we're returning strings for numeric values.